spec/models/course_spec.rb

```diff
@@ -574,22 +459,18 @@
          ': [<filenames of required files for assignment>], required_only: <state>> }' do
```

Comment:
Test: get_required_files - when a course has multiple assignments
Change: added an assignment containing required files, whereas all assignments previously did not contain required files. (Making sure that different cases are covered)